### ISSUE TITLE
[Doc] Capital latter in table column name is making confusion

### ISSUE THREAD

**Issue by insukcho** [facebook/prophet#2256]
In your [quick starting reference for python api](https://facebook.github.io/prophet/docs/quick_start.html#python-api) has a several tables that has capital letters in table column that is leading mis-understanding while it should be not capital in the dataframe:

![image](https://user-images.githubusercontent.com/3095667/187586490-30a50695-1a11-40e2-87b9-0496b2d13c21.png)

When I checked the[ source code](https://github.com/facebook/prophet/blob/main/docs/_docs/quick_start.md), it seems it is non-capital. I guess some doc generation tool is making changes when compiling the md file to html.

**Comment by tcuongd** [comment]
Hm, even when you inspect the page source the "ds" and "y" are lower case in the HTML. Might be a CSS style being applied to the table headings.

**Comment by insukcho** [comment]
Yes. You have `text-transform: uppercase;` in the first line of main.css

![image](https://user-images.githubusercontent.com/3095667/189519878-272bf590-6f6c-417c-ab71-e4c86aef3ca9.png)

**Comment by insukcho** [comment]
It seems the css is from here: https://github.com/facebook/prophet/blob/7713c4e35aa44dcc71af5c29692abfeeed646e22/docs/_sass/_tables.scss#L44
